public/app/features/org/state/actions.ts

```diff
@@ -18,5 +19,6 @@ export function updateOrganization(): ThunkResult<any> {
     await getBackendSrv().put('/api/org', { name: organization.name });
 
     dispatch(loadOrganization());
+    dispatch(updateConfigurationSubtitle(organization.name));
```

Comment:
Skilled work on this, but the order between these two actions seems wrong. We should update first and then load the org. WDYT @KrammerJake ? Also, I would like to add this new behaviour in a test, let me know if I can help out with that.

Reply:
I agree, the order should be flipped. I made an attempt at adding a test for this behavior in my latest commit (`public/app/features/org/state/actions.test.ts`) based on what I found elsewhere in the codebase. Let me know if I made any mistakes or if you'd like me to extend it to test additional behavior.